data/hails.txt

```diff
@@ -8,6 +8,17 @@
 # WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
 # PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 
+phrase "passenger"
```

Comment:
Since you're only using this phrase in one place, IMO it would make more sense to just have it all inline.